packages/common/src/resourceHelpers.ts

```diff
@@ -754,6 +723,33 @@ export function isSupportedFileType(filename: string): boolean {
   );
 }
 
+/**
+ * Gets the thumbnail of an item or group.
+ *
+ * @param storageAuthentication Credentials for the request to the storage
```

Comment:
Could you change to "authentication" please

Reply:
So that the doc matches the code? Why would I want that?  ;-)
Thanks for the catch